source/_posts/2021-02-03-release-20212.markdown

```diff
@@ -11,31 +11,40 @@ categories: Release-Notes
 og_image: /images/blog/2021-02/social.png
 ---
 
-<img src='/images/blog/2021-02/social.png' style='border: 0;box-shadow: none;'>
+<img src='/images/blog/2021-02/social.png' style='border: 0;box-shadow: none; margin-bottom: 20px;'>
```

Comment:
Weird, did we used to add margin bottom in the past?

Reply:
I really don't know...

Reply:
Fixed, will push out in a moment.